tests/test_basic_one_way_send.go

```diff
@@ -0,0 +1,228 @@
+package tests
```

Comment:
do we need to name the file specifically as `test_*.go` isntead of `*_test.go`?

Reply:
Yes. If it was `*_test.go`, then they would be included in `go test`. The `test_` prefix is not necessary, and might be redundant since they're already in package `tests`

Reply:
ye can we remove the `test` prefix